src/server/pbsd_init.c

```diff
@@ -253,7 +253,7 @@ init_server_attrs()
 	resource        *presc = NULL;
 	attribute	attrib;
 	int 		i = 0;
-	char		dflt_log_event[64];
+	char		dflt_log_event[8];
```

Comment:
Please make this 22 like Bhroam suggested

Reply:
ahh... I think I missed his comment. I'll change it